libs/labelbox/src/labelbox/schema/annotation_import.py

```diff
@@ -587,6 +587,20 @@ def parent_id(self) -> str:
         """
         return self.project().uid
 
+    def delete(self) -> None:
```

Comment:
I can't use the deletable inheritance to make this work for this class due to this being a hard delete, not flipping a field to False.